index.html

```diff
@@ -238,13 +238,13 @@ <h3>I'm always interested in hearing about new projects, so if you'd like to cha
         </div>
         <div class="contact-footer">
             <ul>
-                <li><a href="https://github.com/byiringiroscar"><img src="./images/github1.png" alt=""></a></li>
-                <li><a href="https://www.linkedin.com/in/oscar-byiringiro-9baa8313a/"><img src="./images/linkedin1.png" alt=""></a></li>
+                <li><a href="https://github.com/byiringiroscar"><img src="./images/github1.png" alt="github"></a></li>
+                <li><a href="https://www.linkedin.com/in/oscar-byiringiro-9baa8313a/"><img src="./images/linkedin1.png" alt="linkedin"></a></li>
                 <li><a href="#"><img src="./images/hand1.png" alt=""></a></li>
```

Comment:
- The image on line 243 is missing an `alt` text, please fix this to help improve the accessibility of your site 🙏